lib/grinder_tools.dart

```diff
@@ -217,9 +225,11 @@ class TestRunner {
   /// `firefox`, `safari`. [concurrency] controls the number of concurrent test
   /// suites run (defaults to 4). [pubServe] is the port of a pub serve instance
   /// serving `test/`.
-  Future testAsync({String name, String plainName, dynamic platformSelector,
-      int concurrency, int pubServe, RunOptions runOptions}) {
+  Future testAsync({dynamic files, String name, String plainName,
+      dynamic platformSelector, int concurrency, int pubServe,
+      RunOptions runOptions}) {
     return _test.runAsync(_buildArgs(
+        files: files,
```

Comment:
An optional positional parameter would be more consistent with the named command-line test runner and with `Dartfmt` and `Analyzer`, but we'd have to wait until [this DEP](https://github.com/lrhn/dep-optionals) until we could actually make it optional.  If we keep it named, maybe `paths` is more self-documenting that you can use a directory path and not just file paths?